lldb/tools/lldb-dap/README.md

```diff
@@ -290,6 +290,30 @@ The initial repl-mode can be configured with the cli flag `--repl-mode=<mode>`
 and may also be adjusted at runtime using the lldb command
 `lldb-dap repl-mode <mode>`.
 
+#### `lldb-dap custom-event`
```

Comment:
Renamed the event to `lldb-dap send-event` and added a check to prevent this from sending some internal events that should be handled by lldb-dap itself.

Reply:
Thanks, the updated documentation & blocking of internally handled events looks good to me